airbyte-integrations/bases/connector-acceptance-test/tools/acceptance_test_config_migration/create_issues.py

```diff
@@ -0,0 +1,113 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+
+import argparse
+import json
+import logging
+import os
+import subprocess
+import tempfile
+from typing import Any, Text, Dict, Optional
+
+import definitions
+import utils
+from jinja2 import Environment, FileSystemLoader
+
+## Update this line before running the script
+from migrations.strictness_level_migration import config
+
+logging.basicConfig(level=logging.DEBUG)
+environment = Environment(loader=FileSystemLoader(utils.MIGRATIONS_FOLDER))
+
+parser = argparse.ArgumentParser(description="Create issues for a list of connectors from a template.")
+utils.add_dry_param(parser)
+utils.add_connectors_param(parser)
+utils.add_allow_beta_param(parser)
+utils.add_allow_alpha_param(parser)
+
+
+def get_test_failure_logs(definition):
+    test_failure_logs = ""
+    if config.MODULE_NAME == "fail_on_extra_columns":
+        connector_technical_name = definitions.get_airbyte_connector_name_from_definition(definition)
+
+        try:
+            with open(f"{utils.MIGRATIONS_FOLDER}/{config.MODULE_NAME}/test_failure_logs/{connector_technical_name}", "r") as f:
+                for line in f:
+                    test_failure_logs += line
+        except FileNotFoundError:
+            logging.warning(f"Skipping creating an issue for {definition['name']} -- could not find an output file for it.")
+            return
+
+    return test_failure_logs
+
+def get_issue_content(source_definition) -> Optional[Dict[Text, Any]]:
+    issue_title = f"Source {source_definition['name']}: {config.ISSUE_TITLE}"
+
+    template = environment.get_template(f"{config.MODULE_NAME}/issue.md.j2")
+
+    # TODO: Make list of variables to render, and how to render them, configurable
+    issue_body = template.render(
+        connector_name=source_definition["name"], release_stage=source_definition["releaseStage"], test_failure_logs=get_test_failure_logs(source_definition)
+    )
```

Comment:
See hardcoding in `get_test_failure_logs` - we'd want to be able to configure stuff like this too